### PR TITLE
[Bybit] add spot tests

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: tests/exchanges/util/test_exchange_market_status_fixer.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def test_exchange_market_status_fixer_with_str_instead_of_floats(self):
         ).market_status == {
             Ecmsc.PRECISION.value: self._get_precision(None, None, None),
             # replace "plop" and nan with computed numbers
-            Ecmsc.LIMITS.value: self._get_limits(0.05, 1e4, 65.99999999999999, 1111111.1111, 3.3, 11111111111)
+            Ecmsc.LIMITS.value: self._get_limits(0.05, 1e4, None, None, 3.3, 11111111111)
```
Comment (Herklos) [inline tests/exchanges/util/test_exchange_market_status_fixer.py:167]: Isn't it an issue to have None values here?

File: tests/exchanges/util/test_exchange_market_status_fixer.py [hunk 2]
```diff
@@ -197,15 +197,15 @@ def test_exchange_market_status_fixer_without_cost(self):
             Ecmsc.LIMITS.value: self._get_limits(0.05, 1e4, 0.01, 1e4, 0.01 * 0.05, 1e4 * 1e4)
         }
 
-    def test_exchange_market_status_fixer_without_price(self):
+    def test_exchange_market_status_fixer_without_price(self):  #todo
```
Comment (Herklos) [inline tests/exchanges/util/test_exchange_market_status_fixer.py:200]: Still todo ?

File: tests/personal_data/orders/test_decimal_order_adapter.py [hunk 3]
```diff
@@ -290,6 +290,65 @@ async def test_decimal_check_and_adapt_order_details_if_necessary():
...
+    quantity = decimal.Decimal(str(3))
+    price = decimal.Decimal(str(49.9))
+    assert personal_data.decimal_check_and_adapt_order_details_if_necessary(quantity, price, symbol_market) == [
+        (quantity, price)
+    ]
+
+    # quantity too low
+    quantity = decimal.Decimal(str(0.4))
+    price = decimal.Decimal(str(49.9))
+    assert personal_data.decimal_check_and_adapt_order_details_if_necessary(quantity, price, symbol_market) == []
```
Comment (Herklos) [inline tests/personal_data/orders/test_decimal_order_adapter.py:326]: :+1:

File: tests/personal_data/orders/test_decimal_order_adapter.py [hunk 4]
```diff
@@ -290,6 +290,65 @@ async def test_decimal_check_and_adapt_order_details_if_necessary():
...
+
+    # cost too low
+    quantity = decimal.Decimal(str(3))
+    price = decimal.Decimal(str(0.01))
+    assert personal_data.decimal_check_and_adapt_order_details_if_necessary(quantity, price, symbol_market) == []
+
+    # cost too high
+    quantity = decimal.Decimal(str(3))
+    price = decimal.Decimal(str(300))
+    assert personal_data.decimal_check_and_adapt_order_details_if_necessary(quantity, price, symbol_market) == [
```
Comment (Herklos) [inline tests/personal_data/orders/test_decimal_order_adapter.py:343]: :+1:

File: tests_additional/real_exchanges/test_bybit_futures.py [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    ExchangeConstantsTickersColumns as Ectc
+from tests_additional.real_exchanges.real_futures_exchange_tester import RealFuturesExchangeTester
+# required to catch async loop context exceptions
+from tests import event_loop
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+class TestBybitRealExchangeTester(RealFuturesExchangeTester):
```
Comment (Herklos) [inline tests_additional/real_exchanges/test_bybit_futures.py:30]: :+1:

File: tests/exchanges/util/test_exchange_market_status_fixer.py (hunk 1 above)
Comment (GuillaumeDSM) [reply tests/exchanges/util/test_exchange_market_status_fixer.py:167]: It's handled in `decimal_check_and_adapt_order_details_if_necessary`, I added tests as well. When price is None, it means there is no minimum / maximum price

Comment (Herklos) [reply tests/exchanges/util/test_exchange_market_status_fixer.py:167]: Ok Nice :+1:

**Review by Herklos — APPROVED** [review]
Nice improvements on market status fixer!

**Pull request merged**